fabric/fabric19/build.gradle

```
@@ -0,0 +1,45 @@
+plugins {
+    id 'fabric-loom' version '0.12-SNAPSHOT'
```

```suggestion
    id 'fabric-loom'
```
La version est déjà précisée dans le projet parent, et si les versions diffèrent ont a une erreur à la configuration.

De même pour la 1.19.2.